tests/lean/simp_trace.lean

```diff
@@ -81,3 +81,7 @@ example : a ∧ (b ∧ b) ↔ a ∧ b := by simp [my_thm]
 example : (a ∧ (b ∧ b)) = (a ∧ b) := by simp only [my_thm]
 
 example : x - 1 + 1 = x := by simp (discharger := sorry) [Nat.sub_add_cancel]
+
+example (h' : bla x = x) : x + x = x := by
+  simp [bla, h] at *
+  exact h'
```

Comment:
To be thorough, could we also have the tests here at use `at h'`, and another that uses `at *` in which there are actually multiple hypotheses that get simplified, and different simp lemmas are used at each hypothesis?

Reply:
Done. Thanks for the review!